bridge/client/app/_components/ktb-webhook-settings/ktb-webhook-settings.component.html

```diff
@@ -184,11 +184,15 @@ <h2>Webhook configuration</h2>
     </div>
     <ng-template #sendStartedOverlay>
       <p>
-        If you subscribe your webhook to a task event of type triggered, the corresponding started and finished events
-        are sent automatically.
+        Sending started and finished from webhook service is an "async" model. The Keptn task immediately returns. Async
+        is easier at first to implement as it needs no changes to the receiving side, but the task timing does not
+        include the processing time of the receiving system. If you subscribe your webhook to a task event of type
+        triggered, the corresponding started and finished events are sent automatically.
         <br /><br />
-        To move the responsibility of sending the started event to the webhook receiver, select the option 'by webhook
-        receiver'.
+        Delegating the finished event to the receiving service will wait until the receiver is done. Thus it is a
```

Comment:
Could you please change this paragraph to: 

Delegating the stated event to the receiving service will block the task execution until the receiving service sends the started event. However, this allows users to send data from the system back to Keptn and gives accurate timings. To move the responsibility of sending the started event to the webhook receiver, select the option 'by webhook receiver'.